script/test-history

```diff
@@ -9,9 +9,13 @@
 #
 #              Inspired by a Gary Bernhardt script: https://youtu.be/ZQnyApKysg4?t=183
 
+GREEN='\033[1;32m'
```

Comment:
This doesn't look very green in my terminal, but it's not a big deal for me:
<img width="516" alt="image" src="https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/assets/19826940/ac8afc3e-fbfa-45e4-a3a7-ac5c55fd7d3b">

Reply:
Oof. I guess at least it doesn't look red. Do the normal passing test outputs look green? Only difference here should be the bold

Reply:
Yeah they do, bizarre